doc/user-guide-v1.adoc

```diff
@@ -156,7 +156,10 @@ Each `OpenLibertyApplication` CR must specify `.spec.applicationImage` field. Sp
 | `serviceability.size` | [[crd-spec-serviceability-size]] A convenient field to request the size of the persisted storage to use for serviceability. Can be overridden by the `serviceability.volumeClaimName` property.
 | `serviceability.storageClassName` | [[crd-spec-serviceability-storageClassName]] A convenient field to request the StorageClassName of the persisted storage to use for serviceability. Can be overridden by the `serviceability.volumeClaimName` property.
 | `serviceability.volumeClaimName` | [[crd-spec-serviceability-volumeClaimName]] The name of the link:++https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims++[PersistentVolumeClaim] resource you created to be used for serviceability. Must be in the same namespace.
-| `serviceAccountName` | The name of the Red Hat OpenShift service account to be used during deployment. If a service account name is not specified, a service account is automatically created. For examples, see link:#create-a-service-account[Create a service account].
+| `serviceAccountName` | Deprecated. Use link:#crd-spec-serviceAccount-name[`serviceAccount.name`] instead.
+| `serviceAccount`| [[crd-spec-serviceAccount-name]] The name of the service account to be used during deployment. If a service account name is not specified, a service account is automatically created. For examples, see link:#create-a-service-account[Create a service account].
```

Comment:
Suggestion (match the WLO doc):
`The service account to use for application deployment. If a service account name is not specified, a service account is automatically created. For examples, see link:#create-a-service-account[Create a service account].`